src/ol/expr/cpu.js

```diff
@@ -256,7 +256,22 @@ function compileAccessorExpression(expression, context) {
   const name = /** @type {string} */ (nameExpression.value);
   switch (expression.operator) {
     case Ops.Get: {
-      return (context) => context.properties[name];
+      return (context) => {
+        const args = expression.args;
+        let value = context.properties[name];
+        for (let i = 1, ii = args.length; i < ii; ++i) {
+          if (typeof value !== 'object') {
```

Comment:
Should we continue on `null`?

Reply:
Good point, I‘ll add a null check tomorrow.